web/src/common/Loading.svelte

```diff
@@ -20,13 +20,19 @@
 
 {#if loading}
   <div class="cover">
-    {loading}
+    <div class="background">
+      {loading}
 
-    {#if progress != null}
-      <div>
-        <progress value={progress} style:width="100%" />
-      </div>
-    {/if}
+      {#if progress != null}
+        <div>
+          {#if progress == 100}
```

Comment:
When progress is 100 (either because it's set explicitly) or it reaches it from a download, switch to the bounce-back-and-forth-constantly style that just means "something is happening"